### ISSUE TITLE
Is there any way to merge multiple reports into one single report?

### ISSUE THREAD

**Issue by rakeshnambiar** [WasiqB/multiple-cucumber-html-reporter#279]
**Describe your question with as much detail as possible**
Is there any way to merge multiple reports into one single report? Because I am running the whole test suite into multiple batches in parallel to save the execution time. However, this will create individual reports for each job on GitLab. Is there any way for me to combine these reports into one? Has anyone already sorted this out?

Similarly, it would be appreciated if we can display the actual start time and end time in the report rather hard code this value. Pls, guide me on how to do this as well.

**If it's about a specific piece of code, try and include some of it to support your question.**
[...]

**Environment (please complete the following information):**
 - multiple-cucumber-html-reporter: [e.g. 1.10.0]: latest
 - Node.js version: v20.5.
 - NPM version: 9.8.0
 - Platform name and version: OS X
 - Cucumber version: Using cucumber-preprocessor plugin

**Comment by WasiqB** [comment]
Thanks @rakeshnambiar for sharing your findings.

Currently, when multiple Cucumber JSON reports are generated in the single folder, the same gets automatically merged and displayed in the single HTML report.

If in your case, you are generating HTML reports in different jobs and want to merge the same, it is not advisable as the file size may become huge when there are 100s or even 1000s of scenarios. The better approach would be to collect Cucumber JSON reports from all the jobs and use the multiple-cucumber-html-reporter to generate a single report for all these JSON reports.

There's nothing to be done with this issue, hence closing the same.

You can try using the latest version of the reporter and raise a new issue if you find any new observation.
